buttons/buttons.css

```diff
@@ -2165,87 +2170,131 @@ a:focus-visible {
 	transition: 0.4s all ease;
 	color: #000000;
 	border: 1px solid #000000;
-  }
-  
-  /* Jasimrazi-btn-1 ends here */
+}
 
+/* Jasimrazi-btn-1 ends here */
 
 /* cypher's btn-3 start */
 
 .cypher-btn-3 {
-    font-family: Helvetica;
-    /* letter-spacing: 0.2em; */
-    font-weight: 600;
-    font-size: 25px;
-    text-align: center;
-    color: #202125;
-    cursor: pointer;
-    /* max-width: 480px; */
-    width: 59%;
-    height: 30%;
-    outline: 3px solid;
-    outline-color: rgba(71, 126, 232,0.5);
-    outline-offset: 20px;
-    transition: all 200ms cubic-bezier(0.2, 0, 0, 0.8);
+	font-family: Helvetica;
+	/* letter-spacing: 0.2em; */
+	font-weight: 600;
+	font-size: 25px;
+	text-align: center;
+	color: #202125;
+	cursor: pointer;
+	/* max-width: 480px; */
+	width: 59%;
+	height: 30%;
+	outline: 3px solid;
+	outline-color: rgba(71, 126, 232, 0.5);
+	outline-offset: 20px;
+	transition: all 200ms cubic-bezier(0.2, 0, 0, 0.8);
 }
 .cypher-btn-3:hover {
 	color: rgba(71, 126, 232, 0.7);
-  outline-color: rgba(71, 126, 232, 0);
-  outline-offset: 300px;
+	outline-color: rgba(71, 126, 232, 0);
+	outline-offset: 300px;
 }
 
-
-  
 /* cypher's btn-3 end*/
 
 /* Deepesh Sharma's button 1 starts*/
 .deepeshsharmaofficial-button-1 {
-    border: 1px solid #3498db;
-    background: none;
-    padding: 10px 40px;
-    font-size: 20px;
-    font-family: "montserrat";
-    font-weight: bold;
-    cursor: pointer;
-    margin: 10px;
-    transition: 0.8s ease-out;
-    position: relative;
-    overflow: hidden;
-    color: #fff;
-    z-index: 1;
+	border: 1px solid #3498db;
+	background: none;
+	padding: 10px 40px;
+	font-size: 20px;
+	font-family: 'montserrat';
+	font-weight: bold;
+	cursor: pointer;
+	margin: 10px;
+	transition: 0.8s ease-out;
+	position: relative;
+	overflow: hidden;
+	color: #fff;
+	z-index: 1;
 }
 
 .deepeshsharmaofficial-button-1:hover {
-    color: #3498db;
-    font-weight: bold;
+	color: #3498db;
+	font-weight: bold;
 }
 
 .deepeshsharmaofficial-button-1::before {
-    content: "";
-    position: absolute;
-    left: 0;
-    width: 100%;
-    height: 0%;
-    background: #2498db;
-    z-index: -1;
-    transition: 0.8s;
-    top: 0;
-    border-radius: 0 0 50% 50%;
-    height: 180%;
+	content: '';
+	position: absolute;
+	left: 0;
+	width: 100%;
+	height: 0%;
+	background: #2498db;
+	z-index: -1;
+	transition: 0.8s;
+	top: 0;
+	border-radius: 0 0 50% 50%;
+	height: 180%;
 }
 
 .deepeshsharmaofficial-button-1:hover::before {
-    height: 0%;
+	height: 0%;
 }
 /* Deepesh Sharma's button 1 ends*/
 
+/* Jai Marothiya's Button-1 start */
+.Jai-Marothiya-btn-1 {
+	width: 9rem;
+	height: 3.5rem;
+	transition: all 1s;
+	background: blue;
+	box-shadow: rgba(163, 163, 242, 0.25) 0px 8px 14px -4px,
+		rgba(169, 169, 250, 0.25) 0px 5px 8px -5px;
+	border: none;
+	border-radius: 0.3rem;
+}
+
+.Jai-Marothiya-btn-1:hover {
+	background: rgb(235, 78, 125);
+	scale: 1.1;
+	animation: shake 0.5s linear 2;
+}
+
+@keyframes shake {
+	10%,
```

Comment:
also animation key frames should have like your button css class names like jay-marothiya-button-1-shake